testsuite/features/secondary/min_rhlike_appstreams.feature

```diff
@@ -0,0 +1,160 @@
+# Copyright (c) 2024 SUSE LLC
+# Licensed under the terms of the MIT license.
+
+@rhlike_minion
+Feature: Native AppStreams support for Red Hat-like Salt minion
+  
+  Scenario: Log in as admin user
+    Given I am authorized for the "Admin" section
```

Comment:
```suggestion
    Given I am authorized
```

It does not seems you need to be authorized with admin user.
Better use the step I suggested, that way it will generate a new user that will only be used for this particular Cucumber feature. This helps us to better isolate tests, in particular regarding SSM.

Reply:
I missed that step, if you need Admin menu. Then all good :)